docs/getting-started/create-server-spm.html

```diff
@@ -103,6 +103,7 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">Create a server with Swift Package Manager</h1>
+        <p class="block-text">Swift Package Manager (SPM) is included by default with Swift and we will use it to create a new Kitura project from scratch.</p>
```

Comment:
I don't like the wording "and we will use it" I'd prefer something like
"The Swift Package Manager (SPM) is installed as part of Swift by default. In this guide we will use SPM to create a Kitura project from scratch."